example/src/App.js

```diff
@@ -415,6 +417,20 @@ const App = () => {
         {JSON.stringify(inputCheckboxValue)}
       </Section>
 
+      <Section title="Input.Toggle">
+        <h1>Input.Toggle</h1>
+        <Toggle
+          onClick={(e) =>
+            console.log(e.target.checked) ||
```

Comment:
Huh, I just saw Namanda leave this too. I'm guessing y'all both copy/pasted from my `Checkbox` component example.
It won't hurt anything, but you can remove it if you like, and remove the one from checkbox as well.